src/login/controller.ts

```diff
@@ -0,0 +1,30 @@
+import { IsString } from 'class-validator'
```

Comment:
we need to call this folder logins now.. let's see how to change this without creating this extra folder

Reply:
We can delete logins. Keep the login as the file reference in index.ts refers to login.